README.md

```diff
@@ -25,7 +25,9 @@
 Trivy uses `trivy-db` internally to manipulate vulnerability DB. This DB has vulnerability information from NVD, Red Hat, Debian, etc.
 
 ### CLI
-`trivy-db` builds vulnerability DBs on GitHub Actions and uploads them to GitHub Release periodically.
+The `trivy-db` CLI tool builds vulnerability DBs. A [GitHub Actions workflow](.github/workflows/cron.yml)
+periodically builds a fresh version of the vulnerability DB using `trivy-db` and uploads it as a GitHub
+release (see [Download the vulnerability database](#download-the-vulnerability-database) below).
```

Comment:
GitHub release is not used. It uses GHCR.

Reply:
Whoops - fixed, thanks!